shared/src/models/utils/area-statistics.ts

```diff
@@ -0,0 +1,61 @@
+import { IsInt, IsObject, IsPositive } from 'class-validator';
+import type { Vehicle } from '../vehicle';
+import { getCreate } from './get-create';
+import type { PatientStatus } from './patient-status';
+import type { PersonnelType } from './personnel-type';
+
+/**
+ * The statistics for an area in the exercise (e.g. a viewport or the whole exercise).
+ */
+export class AreaStatistics {
+    /**
+     * The number of patients per type
+     */
+    @IsObject()
+    readonly patients: {
+        readonly [visibleStatus in PatientStatus]?: number;
+    };
+
+    /**
+     * The number of vehicles (inclusive in transfer) per type.
+     */
+    @IsObject()
+    readonly vehicles: {
+        readonly [vehicleType: Vehicle['vehicleType']]: number | undefined;
```

Comment:
The same as the other values - but here the optional operator `?` doesn't work (Maybe because vehicleType is of type string and therefore `?` and `undefined` would be the same?)